mesheryctl/internal/cli/root/version.go

```diff
@@ -65,37 +53,54 @@ var versionCmd = &cobra.Command{
 		build := mctlCfg.GetVersion().GetBuild()
 		commitsha := mctlCfg.GetVersion().GetCommitSHA()
 
+		version := cfg.Version{
+			Build:     "unavailable",
+			CommitSHA: "unavailable",
+		}
+
 		logrus.Infof("Client Version: %v \t  GitSHA: %v", build, commitsha)
 
 		req, err := http.NewRequest("GET", fmt.Sprintf("%s/server/version", url), nil)
-		if requestErr(err, url) {
+		if err != nil {
+			logrus.Infof("Server Version: %v \t  GitSHA: %v", version.Build, version.CommitSHA)
+			logrus.Errorf("\nUnable to get request context: %v", err)
 			return
 		}
 
+		defer checkMesheryctlClientVersion(build)
 		client := &http.Client{}
 		resp, err := client.Do(req)
```

Comment:
Nit-pick: As good practice always close the response body `defer resp.Body.Close()`

Reply:
done !